packages/graphql-mini-transforms/rollup.config.mjs

```diff
@@ -5,12 +5,16 @@ export default buildConfig(import.meta.url, {
     './src/index.ts',
     './src/jest.ts',
     './src/jest-simple.ts',
+    './src/rollup-plugin.ts',
     './src/webpack-loader.ts',
+    './src/vite-plugin.ts',
   ],
   entrypoints: {
     index: './src/index.ts',
     jest: './src/jest.ts',
     'jest-simple': './src/jest-simple.ts',
+    rollup: './src/rollup-plugin.ts',
```

Comment:
I wanted to do what you're describing but TS was giving me a hard time on the `rollup` one because it mapped the import for `'rollup'` to that same module. I can rename the entrypoints to include `-plugin` if you'd prefer.